Fails to build. Possibly related to case sensitive file systems?

../QuarkPhysics/qjoint.h:30:10: fatal error: QRigidBody.h: No such file or directory
   30 | #include "QRigidBody.h"
      |          ^~~~~~~~~~~~~~
compilation terminated.

I get a bunch of no file or directory messages trying to build the project. Perhaps my FS is case sensitive? On Garuda Linux, with a BTRFS filesystem.

--- a/QuarkPhysics/qjoint.h
+++ b/QuarkPhysics/qjoint.h
@@ -27,7 +27,7 @@
 
 #ifndef QJOINT_H
 #define QJOINT_H
-#include "QRigidBody.h"
+#include "qrigidbody.h"
 
 
 /**

--- a/examples/examplescenebenchmarkboxes.h
+++ b/examples/examplescenebenchmarkboxes.h
@@ -28,7 +28,7 @@
 #ifndef EXAMPLESCENEBENCHMARKBOXES_H
 #define EXAMPLESCENEBENCHMARKBOXES_H
 
-#include "../QExampleScene.h"
+#include "../qexamplescene.h"
 
 class ExampleSceneBenchmarkBoxes : public QExampleScene
 {

--- a/examples/examplescenebenchmarkboxes2.h
+++ b/examples/examplescenebenchmarkboxes2.h
@@ -28,7 +28,7 @@
 #ifndef EXAMPLESCENEBENCHMARBOXES2_H
 #define EXAMPLESCENEBENCHMARBOXES2_H
 
-#include "../QExampleScene.h"
+#include "../qexamplescene.h"
 
 class ExampleSceneBenchmarkBoxes2 : public QExampleScene
 {

--- a/main.cpp
+++ b/main.cpp
@@ -32,15 +32,15 @@
 #include "qphysicsrenderer.h"
 
 //Example Scenes
-#include "Examples/examplescenebenchmarkboxes.h"
-#include "Examples/examplescenebenchmarkboxes2.h"
-#include "Examples/examplescenejoints.h"
-#include "Examples/examplescenemixedbodies.h"
-#include "Examples/examplesceneraycasts.h"
-#include "Examples/examplescenesoftbodies.h"
-#include "Examples/examplesceneplatformer.h"
-#include "Examples/examplescenefrictions.h"
-#include "Examples/examplesceneblobs.h""
+#include "examples/examplescenebenchmarkboxes.h"
+#include "examples/examplescenebenchmarkboxes2.h"
+#include "examples/examplescenejoints.h"
+#include "examples/examplescenemixedbodies.h"
+#include "examples/examplesceneraycasts.h"
+#include "examples/examplescenesoftbodies.h"
+#include "examples/examplesceneplatformer.h"
+#include "examples/examplescenefrictions.h"
+#include "examples/examplesceneblobs.h""
 
 
 //Font Resource Binary Header

